pandas/_libs/include/pandas/vendored/klib/khash_python.h

```diff
@@ -2,10 +2,14 @@
 #include <Python.h>
 
 
-// use numpy's definitions for complex
-#include <numpy/arrayobject.h>
-typedef npy_complex64 khcomplex64_t;
-typedef npy_complex128 khcomplex128_t;
+typedef struct {
+    float real;
+    float imag;
+    } khcomplex64_t;
```

Comment:
```suggestion
} khcomplex64_t;
```

Just a minor readability thing if you have to edit again. We should probably run this whole file through `clang-format` in a separate PR / CI